ChainSafe/web3.unity
# Lootbox namespace change and error checks

Issue:
We need to start preparing lootboxes for OPM submission, the scope of this task is a namespace change and error checks #660 

Updates:
- Namespace changed from chainlink.lootboxes to lootboxes.chainlink and all occurrences updated.
- Updated the transition in login.cs so that the lootbox scene may be used with the login scene if the user hasn't got the main scene set as their 2nd scene (we'll need this once the contracts are up on chain) [REVERTED]
- We still need to swap all the data so it can use the login scene to be connected to and to pull data from testnet/mainnet as opposed to a local chain, will wait for @oleksandrchainsafe regarding this before we submit to OPM.

To Test:
Import the lootbox package and run it in the editor. You don't need to use the login scene currently as the launcher script initializes from a local rpc node. Just update your web3config class variables in web3.unity\src\UnitySampleProject\Assets\Samples\web3.unity SDK Lootboxes\1.0.0\Chainlink Lootbox Sample\Scripts\ChainlinkLootboxSampleLauncher.cs to match your local node config.

Expected Behavior:
No errors on import due to the namespace changes.
If you're testing with a local RPC node, everything should work as intended.
